pipeline-scripts/buildlib.groovy

```diff
@@ -164,7 +164,9 @@ def setup_venv(use_python38=false) {
         commonlib.shell(script: "rm -rf art-tools/elliott ; cd art-tools; git clone https://github.com/${where[0]}/elliott.git; cd elliott; git checkout ${where[1]}")
     }
 
-    commonlib.shell(script: "pip install -e art-tools/elliott/ -e art-tools/doozer/ -e pyartcd/")
+    commonlib.shell(script: "cd art-tools; rm -rf art-tools; git clone https://github.com/openshift-eng/art-tools.git; cd art-tools; git checkout master")
```

Comment:
How does this work? Are we not having art-tools as a submodule to aos-cd-jobs (sorry if i forgot a previous discussion) ?

Reply:
I like our current pattern of keeping it as a submodule, and let's keep it on the root level so it's not `art-tools/art-tools`?

Reply:
Ahh regarding that, I wanted to keep this PR reversible. If something happens and we need to roll back, we just need to change the paths to the old doozer and elliott submodules. I was thinking if after a couple of releases we don't have any issues, I'll swap in the whole art-tools folder in the root level. What do you think?

Reply:
What will go wrong though? :)
If we've tested things and it works I don't see us rolling back. And if in case we do need to rollback, we'll revert this PR, and add submodules on it - does not sound much effort to me. I would say let's go forward and not think of going back, we fix issues and they come up

Reply:
Makes sense. I was trying to follow a defensive approach, but with the brainpower of our team, I think we can push forward